src/driver/driver.js

```diff
@@ -29,7 +29,7 @@ class Driver {
   static async createInviteDriver(
     data,
     files,
-    { clock, uploadService, token }
+    { clock, uploadService, token, log }
```

Comment:
@coderabbitai change this file to typescript from javascript. Add this in future milestone.